packages/http-server/src/http-server.ts

```diff
@@ -84,14 +91,22 @@ export class HttpServer {
    */
   constructor(
     requestListener: RequestListener,
-    serverOptions?: HttpServerOptions,
+    serverOptions: HttpServerOptions = {},
   ) {
     this.requestListener = requestListener;
     this.serverOptions = serverOptions;
     this._port = serverOptions ? serverOptions.port || 0 : 0;
     this._host = serverOptions ? serverOptions.host : undefined;
     this._protocol = serverOptions ? serverOptions.protocol || 'http' : 'http';
```

Comment:
Since `serverOptions` are always defined, the checks `serverOptions ?:` can be removed now. (This applies to all three lines above.)